app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ActiveRecord::Base
+  has_many :lists
```

Comment:
Should add `dependent: :destroy` to this since wishlists can't exist without a parent event (or shouldn't be able to). To make sure that this can integrate cleanly with current stuff you can use a migration to assign all current wishlists to a dummy event.